app/views/relationships/index.html.haml

```diff
@@ -0,0 +1,21 @@
+%section.people
+  %header
+    %h2 People I Follow
+  %table.table
+    %thead
+      %tr
+        %th(width="30%")
+        %th(width="20%") Videos in Queue
+        %th(width="20%") Followers
+        %th(width="30%") Unfollow
+    %tbody
+      - @relationships.each do |relationship|
+        %tr
+          %td
+            %img(src="http://www.gravatar.com/avatar/#{Digest::MD5.hexdigest(relationship.leader.email.downcase)}?s=40")
```

Comment:
I would prefer to use `= image_tag gravatar_for(relationship.leader.email)` as it looks more succinct. And `gravatar_for` would be a helper method in `ApplicationHelper` as you will need to re-use this code. Also because such complex urls and long char line is better hidden :D